nuxt.config.js

```diff
@@ -23,4 +24,99 @@ c.dir = {
   store:      path.posix.join(SHELL, 'store'),
 };
 
+c.build.extend = (config, { isClient, isDev }) => {
+  // Override the webpack target, so that we get the correct mix of
+  // electron (chrome) + nodejs modules (for ipcRenderer).
+  config.target = 'electron-renderer';
+  // Set a resolver alias for `./@pkg` so that we can load things from @ in CSS
+  config.resolve.alias['./@pkg'] = path.resolve(__dirname, 'pkg', 'rancher-desktop');
+
+  if ( isDev ) {
+    config.devtool = 'cheap-module-source-map';
+  } else {
+    config.devtool = 'source-map';
+  }
+
+  // if ( resourceBase ) {
+  //   config.output.publicPath = resourceBase;
+  // }
+
+  // Remove default image handling rules
+  for ( let i = config.module.rules.length - 1 ; i >= 0 ; i-- ) {
+    if ( /svg/.test(config.module.rules[i].test) ) {
+      config.module.rules.splice(i, 1);
+    }
+  }
+
+  config.resolve.symlinks = false;
+
+  // Ensure we process files in the @rancher/shell folder
```

Comment:
Do you have examples of things we want to force process?

Ah; please moves this to around when we add the custom rule in (so it's clearer)?